charts/temporal/values.yaml

```diff
@@ -373,6 +373,7 @@ admintools:
   securityContext: {}
   podDisruptionBudget: {}
 web:
+  # initContainers: []
```

Comment:
Please make this `additionalInitContainers` to match the API for this functionality for other deployments.